CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - Reexport PAC as `pac` for consistency with other crates, consider `stm32` virtually deprecated
+- Added external interrupt (EXTI) support for output pins to allow using interrupts with open drain
```

Comment:
Uhm, but you're adding the support for all output modes, right? 😅

Reply:
I added that bit to at least try to explain why somebody would want to use interrupts with "output" pins. If you want I can reword it.

I think it'd be better to just remove `InputPin` as well as EXTI from non-OD pins in a future release though.

Reply:
Well, if you add EXTI support for any output mode then that's what the CHANGELOG entry should say.

Reply:
Now it only says that.